### DISCUSSION TITLE
Cannot make any changes to templates or services

### DISCUSSION THREAD

**Discussion by FurdyHrosek** [emailjs-com/help#97] [Q&A] [answered]
Hi!

Got a problem here... :( No matter what I do, I cannot change any template or service. This is the 2nd day its happening.

The account is under mhoudek11@gmail.com and public key is 17QY5w2jr6Pk9ULOU

Please see the video attached.

Thanks!
Martin

https://github.com/emailjs-com/help/assets/75339000/457d9015-6a57-49f4-ab31-319247deb806

**Comment by xr0master** [comment]
Try to refresh the page (or relogin). Maybe your session is expired.

**Reply by FurdyHrosek** [reply]
Didn't help, cleared the cache etc.

**Reply by xr0master** [reply, marked as answer]
You can also try the incognito mode since you have a lot of browser extensions.

**Reply by FurdyHrosek** [reply]
You are absolutely right, my bad :D Thanks!

**Comment by evilgeniuscreative** [comment]
Well, that's pretty garbage UI if you need to use the incognito window to FILL OUT A FORM or modify it. C'mon guys, you can do better than that, emailjs.

**Reply by xr0master** [reply]
Hm.. By speaking in your language, you turn your browser into a garbage can that can't do basic things, but you expect any UI to work perfectly there.  

The Incognito mode simply returns basic functionality to the browser.

**Reply by evilgeniuscreative** [reply]
Don't be silly there's no reason to have to do it in it incognito. Also in incognito whatever garbage I put on there you and I both know doesn't even come up ...
